benchmark_terraform/modules/benchmark/service/run_benchmark.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+## Enable strict error handling, command tracing, and pipefail
+set -euxo pipefail
+
+# 1. Configure s3cmd
+# 2. Run benchmark
+# 3. Upload benchmark results to s3 bucket
+
+## Configure s3cmd
+s3cmd --dump-config \
+    --host="$BENCHMARK_ENDPOINT" \
+    --host-bucket="%(bucket)s.$BENCHMARK_ENDPOINT" \
+    --access_key="$AWS_ACCESS_KEY_ID" \
+    --secret_key="$AWS_SECRET_ACCESS_KEY" \
+    --multipart-chunk-size-mb=4096 > ~/.s3cfg
+
+## Run actual benchmark
+ruby bench.rb --chain calibnet --tempdir ./snapshots --daily
+ruby bench.rb --chain mainnet --tempdir ./snapshots --daily
+
+## Upload benchmark result to s3 weekly file
+week_number=$(date +%W) # Week starting on Monday
+s3cmd get s3://"$BENCHMARK_BUCKET"/benchmark-results/weekly-results/weekly_result_"$week_number".csv /tmp/weekly_result_"$week_number".csv --force || 
```

Comment:
what is the point of `||` at the end?
```suggestion
s3cmd get s3://"$BENCHMARK_BUCKET"/benchmark-results/weekly-results/weekly_result_"$week_number".csv /tmp/weekly_result_"$week_number".csv --force
```